store/profile.ts

```diff
@@ -159,15 +160,6 @@ export const actions: ActionTree<State, RootState> = {
       commit('scholars/insertScholar', updatedScholar, { root: true })
     }
 
-    if (updatedPrivate) {
-      let updatedScholar = Scholar.clone(scholar)
-      updatedScholar.scholarPrivate = {
-        recordName: updatedPrivate.recordName!,
-        action: CloudKit.ReferenceAction.DELETE_SELF
-      }
-      await updatedScholar.save()
-    }
-
```

Comment:
I think we still need this in case the scholar didn't have a ScholarPrivate record before saving.

Maybe the check to save should be something like `updatedPrivate.recordName !== getters.scholar?.scholarPrivate.recordName` instead 🤔

Reply:
I think the problem with that is that [the email doesn't get saved](https://github.com/WWDCScholars/web-app/blob/5201131fec299fb2a8b0aecb05e952097e47fdf8/store/profile.ts#L127) if you **only** change the email. Just leaving it in often results in double saves which causes conflicts, which *works* but doesn't look nice..

```
_ckErrorCode: "CONFLICT"
_reason: "client oplock error updating record"
_recordName: "A6D87629-********"
_serverErrorCode: "CONFLICT"
message: "client oplock error updating record"
```